packages/flutter/test/widgets/platform_view_test.dart

```diff
@@ -1016,6 +1016,541 @@ void main() {
     });
   });
 
+  group('Android PlatformView', () {
```

Comment:
```suggestion
  group('AndroidViewSurface', () {
```

Reply:
Is there any difference in behavior between `AndroidView` and `AndroidViewSurface`?